### PR TITLE
Add clr metrics scope and oal script

### PR DESCRIPTION
**Pull request by liuhaoyang**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
https://github.com/SkyAPM/SkyAPM-dotnet/issues/108
___
### New feature or improvement
- Describe the details and related test reports.
As part of the CLR Metrcis feature, I added three scopes: `ServiceInstanceCLRGC`, `ServiceInstanceCLRGC`, and `ServiceInstanceCLRThread`, and the corresponding oal script.

### CODE REVIEW

File: oap-server/generated-analysis/src/main/resources/official_analysis.oal [hunk 1]
```diff
@@ -79,4 +79,15 @@ database_access_p99 = from(DatabaseAccess.latency).p99(10);
...
\ No newline at end of file
+database_access_p50 = from(DatabaseAccess.latency).p50(10);
+
+// CLR instance metric
+instance_clr_cpu = from(ServiceInstanceCLRCPU.usePercent).doubleAvg();
+instance_clr_gen0_collect_count = from(ServiceInstanceCLRGC.gen0CollectCount).sum();
+instance_clr_gen1_collect_count = from(ServiceInstanceCLRGC.gen1CollectCount).sum();
+instance_clr_gen2_collect_count = from(ServiceInstanceCLRGC.gen2CollectCount).sum();
+instance_clr_heap_memory = from(ServiceInstanceCLRGC.heapMemory).longAvg();
+instance_clr_available_completion_port_threads = from(ServiceInstanceCLRThread.availableCompletionPortThreads).count();
```
Comment (wu-sheng) [inline oap-server/generated-analysis/src/main/resources/official_analysis.oal]: The count of thread number? I think should you want new indicator? `max` or somthing?

Comment (wu-sheng) [reply oap-server/generated-analysis/src/main/resources/official_analysis.oal]: Same concern about the following thread related metric.

Comment (liuhaoyang) [reply oap-server/generated-analysis/src/main/resources/official_analysis.oal]: This is an indicator of the CLR thread pool. Used to observe available I/OCP threads and worker threads  https://docs.microsoft.com/zh-cn/dotnet/api/system.threading.threadpool.getavailablethreads?view=netstandard-2.0

Comment (wu-sheng) [reply oap-server/generated-analysis/src/main/resources/official_analysis.oal]: Yes, I know roughly what it is. I just say, the indicator is not right.

File: oap-server/generated-analysis/src/main/resources/official_analysis.oal [hunk 2]
```diff
@@ -79,4 +79,15 @@ database_access_p99 = from(DatabaseAccess.latency).p99(10);
...
+// CLR instance metric
+instance_clr_cpu = from(ServiceInstanceCLRCPU.usePercent).doubleAvg();
+instance_clr_gen0_collect_count = from(ServiceInstanceCLRGC.gen0CollectCount).sum();
+instance_clr_gen1_collect_count = from(ServiceInstanceCLRGC.gen1CollectCount).sum();
+instance_clr_gen2_collect_count = from(ServiceInstanceCLRGC.gen2CollectCount).sum();
+instance_clr_heap_memory = from(ServiceInstanceCLRGC.heapMemory).longAvg();
+instance_clr_available_completion_port_threads = from(ServiceInstanceCLRThread.availableCompletionPortThreads).max();
+instance_clr_available_worker_threads = from(ServiceInstanceCLRThread.availableWorkerThreads).max();
+instance_clr_max_completion_port_threads = from(ServiceInstanceCLRThread.maxCompletionPortThreads).max();
+instance_clr_max_worker_threads = from(ServiceInstanceCLRThread.maxWorkerThreads).max();
```
Comment (wu-sheng) [inline oap-server/generated-analysis/src/main/resources/official_analysis.oal]: cc @TinyAllen These are the new metrics for instance, we could add those in RocketBot directly.

**Pull request merged**